test/Microsoft.AspNet.Routing.Tests/project.json

```diff
@@ -1,22 +1,23 @@
 {
-    "compilationOptions": {
```

Comment:
It looks to me like most repos use 4 spaces per indent in JSON files. At least, in the few places I looked.